cps/kobo_sync_status.py

```diff
@@ -54,10 +54,10 @@ def remove_synced_book(book_id, all=False, session=None):
 def change_archived_books(book_id, state=None, message=None):
     archived_book = ub.session.query(ub.ArchivedBook).filter(and_(ub.ArchivedBook.user_id == int(current_user.id),
                                                                   ub.ArchivedBook.book_id == book_id)).first()
-    if not archived_book:
+    if not archived_book and state == True:
```

Comment:
Makes it so only requests to archive (ie state = true) actually creates an archivedbook listing